### ISSUE TITLE
Incorrect typings with this extension and WDIO + Jquery in VSCode

### ISSUE THREAD

**Issue by ntinkler-calendly** [nrwl/nx-console#1346]
## Current Behavior
When I have this extension installed, and I have configured a TS library to use WDIO (and NOT jquery), but I have jquery in top level package.json

Then I always see hover tooltips/errors insisting that `$` is Jquery static.  When that identifier is NOT jquery.  It is the selector from WDIO.

## Expected Behavior
I expect to see the proper type for WDIO typings. I do not expect to see the JQuery typings anywhere, I am not including them in any tsconfig.json (or tsconfig.base.json) and I have them explicitly disabled in VSCode.

This was working as expected until the latest update to the extension on 9/12/22

## Steps to Reproduce

tsconfig.base.json
```
{
  "compileOnSave": false,
  "compilerOptions": {
    "rootDir": ".",
    "sourceMap": true,
    "declaration": false,
    "moduleResolution": "node",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "importHelpers": true,
    "target": "es2015",
    "module": "esnext",
    "lib": ["es2017", "dom"],
    "skipLibCheck": true,
    "skipDefaultLibCheck": true,
    "baseUrl": ".",
    "paths": {
      "@client-core/assets": ["libs/assets/src/index.ts"],
      "@client-core/gcal": ["libs/gcal/src/index.ts"],
      "@client-core/legacy-client-core": [
        "libs/legacy-client-core/src/index.ts"
      ],
      "@client-core/legacy-client-core/bg": [
        "libs/legacy-client-core/src/bgIndex.ts"
      ],
      "@client-core/platform": ["libs/platform/src/index.ts"],
      "@client-core/platform-chrome": ["libs/platform-chrome/src/index.ts"],
      "@client-core/platform-firefox": ["libs/platform-firefox/src/index.ts"],
      "@client-core/shared": ["libs/shared/src/index.ts"],
      "@client-core/webext-wdio-specs": ["libs/webext-wdio-specs/src/index.ts"]
    }
  },
  "exclude": ["node_modules", "tmp"]
}
```

library tsconfig.json
```
{
  "extends": "../../tsconfig.base.json",
  "compilerOptions": {
    "module": "commonjs",
    "esModuleInterop": true,
    "resolveJsonModule": true,
    "sourceMap": true,
    "outDir": "../../dist/out-tsc",
    "target": "ES5",
    "types": ["webdriverio/async", "node", "@wdio/jasmine-framework"]
  },
  "typeAcquisition": {
    "exclude": ["jquery"]
  },
  "include": ["src/**/*.ts", "src/**/*.js"]
}
```

package.json dependencies
```
  "devDependencies": {
    "@nrwl/cli": "14.1.2",
    "@nrwl/cypress": "14.1.2",
    "@nrwl/eslint-plugin-nx": "14.1.2",
    "@nrwl/jest": "14.1.2",
    "@nrwl/linter": "14.1.2",
    "@nrwl/react": "14.1.2",
    "@nrwl/web": "14.1.2",
    "@nrwl/workspace": "14.1.2",
    "@testing-library/jest-dom": "^5.16.4",
    "@testing-library/react": "13.1.1",
    "@types/adm-zip": "^0.5.0",
    "@types/analytics-node": "^3.1.9",
    "@types/jest": "27.4.1",
    "@types/jquery": "^3.5.14",
    "@types/luxon": "^2.3.1",
    "@types/node": "16.11.7",
    "@types/react": "18.0.8",
    "@types/react-dom": "18.0.3",
    "@types/react-is": "17.0.3",
    "@types/react-lazyload": "^3.2.0",
    "@types/redux-mock-store": "^1.0.3",
    "@types/styled-components": "5.1.25",
    "@types/uuid": "^8.3.4",
    "@types/webextension-polyfill": "^0.8.3",
    "@typescript-eslint/eslint-plugin": "~5.18.0",
    "@typescript-eslint/parser": "~5.18.0",
    "@wdio/cli": "^7.19.5",
    "@wdio/firefox-profile-service": "^7.19.5",
    "adm-zip": "^0.5.9",
    "babel-jest": "27.5.1",
    "babel-plugin-styled-components": "1.10.7",
    "chromedriver": "105.0.0",
    "clipboardy": "2.3.0",
    "cypress": "^9.1.0",
    "eslint": "~8.12.0",
    "eslint-config-prettier": "8.1.0",
    "eslint-plugin-cypress": "^2.10.3",
    "eslint-plugin-import": "2.26.0",
    "eslint-plugin-jsx-a11y": "6.5.1",
    "eslint-plugin-react": "7.29.4",
    "eslint-plugin-react-hooks": "4.5.0",
    "eslint-plugin-wdio": "^7.19.4",
    "fs-extra": "^10.1.0",
    "geckodriver": "^3.0.1",
    "git-rev-sync": "^3.0.2",
    "jest": "27.5.1",
    "mockzilla": "^0.12.0",
    "mockzilla-webextension": "^0.13.0",
    "nx": "14.1.2",
    "prettier": "^2.5.1",
    "react-test-renderer": "18.1.0",
    "ts-jest": "27.1.4",
    "ts-node": "9.1.1",
    "tsconfig-paths": "^4.0.0",
    "typescript": "~4.6.2",
    "wdio-chromedriver-service": "^7.3.2",
    "wdio-geckodriver-service": "^2.1.1"
  },
  "dependencies": {
    "@airbrake/browser": "^2.1.7",
    "@nrwl/devkit": "14.1.2",
    "@optimizely/optimizely-sdk": "^4.9.1",
    "@reduxjs/toolkit": "^1.8.1",
    "@segment/analytics-next": "^1.38.0",
    "@types/chrome": "^0.0.190",
    "@types/estree": "^0.0.47",
    "@types/office-js": "^1.0.253",
    "@types/office-runtime": "^1.0.23",
    "@wdio/jasmine-framework": "^7.19.5",
    "@wdio/local-runner": "^7.19.5",
    "@wdio/spec-reporter": "^7.19.5",
    "@wdio/types": "^7.19.5",
    "analytics-node": "^6.1.0",
    "body-parser": "^1.20.0",
    "classnames": "^2.3.1",
    "cookie-parser": "^1.4.6",
    "core-js": "^3.6.5",
    "cors": "^2.8.5",
    "express": "^4.17.3",
    "jquery": "^3.6.0",
    "luxon": "^2.3.2",
    "moment-timezone": "^0.5.34",
    "normalizr": "^3.6.2",
    "react": "18.1.0",
    "react-dom": "18.1.0",
    "react-is": "18.1.0",
    "react-lazyload": "^3.2.0",
    "react-redux": "7.2.8",
    "redux-mock-store": "^1.5.4",
    "regenerator-runtime": "0.13.7",
    "styled-components": "5.3.5",
    "tippy.js": "^6.3.7",
    "tslib": "^2.0.0",
    "underscore": "^1.13.2",
    "uuid": "^8.3.2",
    "webdriverio": "^7.19.5",
    "webext-redux": "^2.1.9",
    "webextension-polyfill": "^0.8.0"
  }
```

### Failure Logs / Images / Videos

Expected typings with extension disabled:
<img width="1652" alt="Screen Shot 2022-09-16 at 10 48 19 AM" src="https://user-images.githubusercontent.com/44983149/190667096-44ab8d54-ab24-48fc-843c-d4f207464432.png">

Typings with this extension enabled:
<img width="1652" alt="Screen Shot 2022-09-16 at 10 49 19 AM" src="https://user-images.githubusercontent.com/44983149/190667302-7615cda2-7d36-40fb-a4af-c53a2ec9fba0.png">

**Comment by ntinkler-calendly** [comment]
I'll add - The project builds just fine.  The issue is only the display in VSCode.

**Comment by Cammisuli** [comment]
thats interesting.. I'll see if i can reproduce it with the information you provided.

**Comment by ntinkler-calendly** [comment]
@Cammisuli appreciate you taking a look!  If it helps, I have some free time this afternoon and I'll to cut my project down to a minimal reproduction I can point you at.

**Comment by Cammisuli** [comment]
Oh perfect, I appreciate that 😊

**Comment by ntinkler-calendly** [comment]
Ok - While trying to cut down a repro package, I have some additional info.

I have a neighbor library that is importing jquery: `import $ from 'jquery';`

If these imports are removed from the other library, the problem goes away.  There are no dependencies between them, and they are full NX libs (complete with their own tsconfig.json files).

I may also be wrong about the previous version having the correct behavior - we refactored some legacy code that was using jquery on 9/13 into a lib from an app, and I assumed the update to the extension on 9/12 was causing the issue, but it looks like it was introducing an import for jquery instead. 

Removing/disabling the extension still causes VSCode to use the correct typings, so something is going on.

I have attached a reproduction project (it's definitely no longer runnable, since I have whacked a lot of things - including all the apps - but it displays the incorrect type hint behavior in vscode.)

To use - you'll need 
nodejs 16.13.2
yarn 1.22.15

run `yarn` at the project root, and then open `libs/webext-wdio-specs/src/lib/specs/dropdown.e2e.ts` is vscode.  Locally - I see incorrect type hints for jquery.

Removing the import in `libs/jqimporter/src/index.ts` causes the issue to disappear.  As does disabling this extension.

[client_core_nx_repro.zip](https://github.com/nrwl/nx-console/files/9586237/client_core_nx_repro.zip)

I also took a short video:
https://monosnap.com/file/AreRJfJIxIl4O4HyRVNgoN7ceismxl
